codalab/rest/bundles.py

```diff
@@ -469,6 +470,65 @@ def _fetch_bundle_location(bundle_uuid: str, bundle_store_uuid: str):
     return BundleLocationListSchema(many=False).dump(bundle_location).data
 
 
+@get('/bundle_stores', apply=AuthenticatedProtectedPlugin())
+def _fetch_bundle_stores():
+    """
+    Fetch the bundle stores available to the user.
+
+    Returns a dictionary in which the keys are the bundle store uuids, and the values
+    are tuples with the owner_id, name, and url.
+    """
+    bundle_stores = local.model.get_bundle_stores(request.user)
+    return BundleStoreSchema(many=True).dump(bundle_stores).data
+
+
+@post('/bundle_stores', apply=AuthenticatedProtectedPlugin())
+def _add_bundle_store():
+    """
+    Add a bundle store that the user can access. Request body must contain the fields in BundleStoreSchema.
```

Comment:
We can't just reference "BundleStoreSchema" in these docstrings, because these docstrings are directly copied over to the REST API reference and the user won't know what to look at. We should document each parameter, and it's okay if it ends up being duplicative across docstrings.